lib/gui/components/drive-selector/templates/drive-selector-modal.tpl.html

```diff
@@ -50,7 +47,17 @@ <h4 class="list-group-item-heading">{{ drive.description }} - {{ drive.size | gi
 </div>
 
 <div class="modal-footer">
-  <button class="button button-primary button-block"
+  <p class="modal-footer-label"
+    ng-show="modal.getButtonStatus() === 'danger'">
+      {{ modal.getFooterMessage() }}
```

Comment:
Since this function is only being used here, and the logic is quite simple, we should be able to inline it. As a bonus, in my comment on that function, I highlighted that the logic could be reduced to simply checking if the button status is `danger`, which you're already doing in the `ngShow` right above, you can put the message here and that should be it.